pkgs/development/python-modules/frida-python/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkgs,
+  typing-extensions,
+}: let
+  pname = "frida-python";
+  version = "16.0.19";
+  namePypi = "frida";
+
+  devkit = fetchurl {
+    url = "https://github.com/frida/frida/releases/download/${version}/frida-core-devkit-${version}-linux-${stdenv.hostPlatform.linuxArch}.tar.xz";
+    hash = "sha256-yNXNqv8eCbpdQKFShpAh6rUCEuItrOSNNLOjESimPdk=";
+  };
+in
+  pkgs.python3Packages.buildPythonPackage rec {
+    inherit pname version;
+
+    src = pkgs.python3Packages.fetchPypi {
+      pname = namePypi;
+      inherit version;
+      hash = "sha256-rikIjjn9wA8VL/St/2JJTcueimn+q/URbt9lw/+nalY=";
+    };
+
+    postPatch = ''
+      mkdir assets;
+      pushd assets
+      tar xvf ${devkit}
+      export FRIDA_CORE_DEVKIT=$(pwd)
+      popd
+    '';
+
+    propagatedBuildInputs = [
+      typing-extensions
+    ];
+
+    pythonImportsCheck = ["frida"];
+
+    meta = with lib; {
+      description = "Dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers (Python bindings)";
+      homepage = "https://www.frida.re";
+      license = licenses.wxWindows;
+      maintainers = with maintainers; [s1341];
```

Comment:
formatting
```suggestion
      maintainers = with maintainers; [ s1341 ];
```

Reply:
Alejandra and nix-fmt result in different formattings. Apologies.